adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -148,13 +179,18 @@ trait AkkaHttpAdapter {
                       subscriptions
                         .modify(map => (map.get(msg.id), map - msg.id))
                         .flatMap(fiber => IO.whenCase(fiber) { case Some(fiber) => fiber.interrupt })
+                        .map { _ =>
+                          keepAliveFiberRef.map(_.map(_.interrupt))
+                          keepAliveFiberRef.set(None)
```

Comment:
This is stopping a single subscription, we don't want to interrupt the keep alive here.